airflow/providers/amazon/aws/example_dags/example_rds.py

```diff
@@ -0,0 +1,126 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This is an example dag for using `RedshiftSQLOperator` to authenticate with Amazon Redshift
+then execute a simple select statement
+"""
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.rds import (
+    RDSCancelExportTaskOperator,
+    RDSCopyDBSnapshotOperator,
+    RDSCreateDBSnapshotOperator,
+    RDSCreateEventSubscriptionOperator,
+    RDSDeleteDBSnapshotOperator,
+    RDSDeleteEventSubscriptionOperator,
+    RDSStartExportTaskOperator,
```

Comment:
Writing it here as it's the easiest.. same comment as in the hook PR
lets use `Rds` not `RDS`

Reply:
Am I misremembering @eladkal or isn't it supposed to be that abbreviatoins are all caps so like `SQL` not Sql` etc?   Or is it the  other way around?  Sorry, I know that this has come up a few times before.

Reply:
Specifically for AWS we discussed it in https://github.com/apache/airflow/issues/20296
The larger scope is discussed in AIP 21

For SQL we actually do use Sql see [SqlSensor](https://github.com/apache/airflow/blob/b38391e2f91760e64576723c876341f532a6ee2d/airflow/sensors/sql.py#L28)

Reply:
we can find examples on both sides of this.  And API-21 discusses import paths but not class case.

and, it seems PEP-8 suggests all-upper case for this kind of thing (see [SO post here](https://stackoverflow.com/a/2853609/2761682)).

And this quote is offered:

> Note: When using abbreviations in CapWords, capitalize all the letters of the abbreviation. Thus HTTPServerError is better than HttpServerError.

HOWEVER, I personally like `Sql` better than `SQL` because of readability... and I for one say "sequel" not "SQL".  (so i support your approach)

An example on this side is `class NASAJPL`... crap readability IMO --> `class NasaJPL` better or `NasaJpl`

We should probably make an AIP that addresses this once and for all, and perhaps enforce with pre-commit.

WDYT?

Reply:
Case 6 in AIP mentions the names.
I agree it's not 100% which is why we are dealing with it months after the AIP wss completed :)

But specifically for AWS We already have convention (Similar to the convention for google provider)
If you feel it should be diffrent i guess it's important enough to take it to the mailing list.

Reply:
It (AIP-21) happens to name a few classes that happen to have abbreviations but the naming changes have nothing to do with the abbreviations and the question of what to do with abbreviations / acronyms / initializations (whatever you want to call it) is not addressed at all.

And it is very easy to find both styles in our repo.  E.g. `CloudSQLBaseOperator`, `BaseSQLOperator`, `BaseSQLToGCSOperator`, `BranchSQLOperator`.  

Interestingly, `BranchSqlOperator` was _deprecated_ and renamed to `BranchSQLOperator` -- which is actually opposite the direction you are going with AWS renames.

So it seems it's all over the place really, and yeah probably we should have a mailing list discussion.

Reply:
I agree that it's not simple to find the right answer.
I'd be happy to get it sorted in the mailing list as the issue is bigger than just this specific operator.

Reply:
Yup, I'll appreciate your input (i created a discussion for this on the list)